.travis.yml

```diff
@@ -4,8 +4,8 @@ python:
   - 3.5
   - "nightly"
 install:
-  - pip install -r requirements-tests.txt
+  - pip install --upgrade -r requirements-tests.txt
 script:
-  - coverage run --source xonsh -m py.test -q
+  - py.test --flake8 --cov=. --cov-report=term
```

Comment:
or we should hard code what we really want to test, we can always use more `--cov` arguments if we test stuff outside the xonsh package dir at a later time, seems easier than adding lines in `ommit:` each time we add something new to the root xonsh dir. The decision is yours though and i'm happy with either way!